package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "ehub-collector",
-  "version": "1.4.2",
+  "version": "1.4.3",
   "dependencies": {
     "@alertlogic/al-azure-collector-js": "2.1.0",
     "@alertlogic/al-collector-js": "1.3.2",
     "@azure/arm-eventhub": "3.2.0",
     "@azure/arm-monitor": "6.0.0",
     "async": "2.6.1",
     "moment": "2.24.0",
```

Comment:
do we need `moment` dependency?

Reply:
yes, moment-timezone has the dependency of the moment.